dag/migrated.yml

```diff
@@ -16,3 +16,7 @@ steps:
   - data://garden/education/2017-09-30/public_expenditure
   data://garden/education/2017-09-30/public_expenditure:
   - snapshot://education/2017-09-30/public_expenditure.feather
+  data://grapher/agriculture/2017-03-08/share_of_agriculture_in_gdp:
```

Comment:
I think it would be better to use the version "2023-10-04", for consistency. Would that be a problem?